api/ngrok/v1alpha1/kubernetesoperator_types.go

```diff
@@ -51,11 +51,6 @@ type KubernetesOperatorDeployment struct {
 }
 
 type KubernetesOperatorBinding struct {
-	// Name is the name of the k8s-binding for the account to bind to this configuration and the ngrok API
-	// +kubebuilder:validation:Required
-	// +kubebuilder:validation:Pattern=`^k8s[/][a-zA-Z0-9-]{1,63}$`
-	Name string `json:"name,omitempty"`
-
```

Comment:
I wonder if we shouldn't leave this field in here but mark it as deprecated and change the validation to optional. The only thing I'm worried about if we remove it right now is if someone deploys this change and then rolls back to a previous version, what happens?

If there aren't a lot of users using bindings right now, I think its a lower risk change.

Reply:
Since we're breaking the API on the hosted side bindings won't work with this field anyway so I don't think it matters _too_ much. Whenever we cut a release with these and a couple other upcoming changes that'll be the first release that works with bindings essentially